src/views/Send/SendNFT.vue

```diff
@@ -0,0 +1,631 @@
+              <div>
+                <h3>{{ selectedNFT.name }}</h3>
+                <p>{{ selectedNFT.collection.name }}</p>
+              </div>
+            </div>
+            <div class="selected-nft-asset__send-details">
+              <h3 class="text-uppercase">Network speed/fee</h3>
+              <div class="d-flex justify-content-between">
+                <p>{{ prettyFee }} {{ asset }}</p>
+                <p>{{ totalFeeInFiat }} USD</p>
+              </div>
+              <div class="form-group mt-4">
+                <h3 for="address" class="text-uppercase text-muted">Send to</h3>
+                <p class="address">
+                  <span class="font-weight-bold">{{ startAddress }}</span
+                  >{{ middleAddressPart }}<span class="font-weight-bold">{{ endAddress }}</span>
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="button-group">
+            <button class="btn btn-light btn-outline-primary btn-lg" @click="next('selectedAsset')">
+              Edit
+            </button>
+            <button class="btn btn-primary btn-lg btn-icon" @click="sendNFT" :disabled="loading">
+              <SpinnerIcon class="btn-loading" v-if="loading" />
+              <template v-else>Send NFT</template>
+            </button>
+          </div>
+        </div>
+      </div>
+    </template>
+    <OperationErrorModal
+      :open="sendErrorModalOpen"
+      :account="account"
+      @close="closeSendErrorModal"
+      :error="sendErrorMessage"
+    />
+  </div>
+</template>
+
+<script>
+import { mapState, mapGetters, mapActions } from 'vuex'
+import NavBar from '@/components/NavBar.vue'
+import { applyActivityFilters } from '@liquality/wallet-core/dist/utils/history'
+import amplitude from 'amplitude-js'
+import Accordion from '@/components/Accordion.vue'
+import { chains } from '@liquality/cryptoassets'
+import { shortenAddress } from '@liquality/wallet-core/dist/utils/address'
+import cryptoassets from '@liquality/wallet-core/dist/utils/cryptoassets'
+import CopyIcon from '@/assets/icons/copy.svg'
+import { getSendFee, getFeeLabel } from '@liquality/wallet-core/dist/utils/fees'
+import { getFeeAsset, getNativeAsset } from '@liquality/wallet-core/dist/utils/asset'
+import { getAssetIcon } from '@/utils/asset'
+import SpinnerIcon from '@/assets/icons/spinner.svg'
+import OperationErrorModal from '@/components/OperationErrorModal'
+import FeeSelector from '@/components/FeeSelector'
+import CustomFees from '@/components/CustomFees'
+import CustomFeesEIP1559 from '@/components/CustomFeesEIP1559'
+import DetailsContainer from '@/components/DetailsContainer'
+import {
+  formatFiat,
+  formatFiatUI,
+  prettyBalance,
+  prettyFiatBalance
+} from '@liquality/wallet-core/dist/utils/coinFormatter'
+import _ from 'lodash'
+import BN from 'bignumber.js'
+import NFTThumbnailImage from '@/assets/nft_thumbnail.png'
+
+amplitude.getInstance().init('bf12c665d1e64601347a600f1eac729e')
```

Comment:
What was this magic value `'bf12c665d1e64601347a600f1eac729e'`?
Should it be part of some wallet-core configuration?